lightning/glue/glue.go

```diff
@@ -48,6 +51,39 @@ type SQLExecutor interface {
 	Close()
 }
 
+// sqlConnSession implement checkpoints.Session used only for lighting itself
+type sqlConnSession struct {
+	checkpoints.Session
+	conn *sql.Conn
+}
+
+func (session *sqlConnSession) Close() {
+	session.conn.Close()
+}
+
+func (session *sqlConnSession) Execute(ctx context.Context, sql string) ([]sqlexec.RecordSet, error) {
+	_, err := session.conn.ExecContext(ctx, sql)
+	return nil, err
+}
+
+func (session *sqlConnSession) CommitTxn(context.Context) error {
+	return errors.New("sqlConnSession doesn't have a valid CommitTxn implementation")
+}
+
+func (session *sqlConnSession) RollbackTxn(context.Context) {}
+
+func (session *sqlConnSession) PrepareStmt(sql string) (stmtID uint32, paramCount int, fields []*ast.ResultField, err error) {
```

Comment:
and return `errors.New` for these methods

Reply:
Todo

Reply:
Done